### PR TITLE
Added conditional check in bubble sort.

### LINKED ISSUE #5: Improve Bubble sort.
Unnecessary iterations even after it gets sorted.

**How to reproduce**
Set Shuffle to ascending.
Now in the first iteration we get to know that, given input is sorted, so what is the need to iterate further.
Or was it intentional?

**Expected Behaviour**
once input gets sorted, it should stop iterating further.

I would like to work on it.

### PR DESCRIPTION
**Pull request by rupav**
Fixes #5 
To avoid iterating even after sorting.

### CODE REVIEW

**Review by caspervonb — CHANGES_REQUESTED** [review]
A few minor nitpicks, code could be leaner.

**Comment by rupav** [comment]
@caspervonb changed as requested. But I am not able to align properly the changes,
I am changing code on sublineText 3 which shows correct alignment,
but then after
`git push -f origin` , alignment gets disturbed.
What to do here?

**Comment by caspervonb** [comment]
@rupav Looks like you have a mix of tabs and spaces! e.g start of line https://github.com/rupav/toneofsorting/blob/10be93847bb204ce8035ebfee70c36fd75511718/sort.js#L19 actually contains two spaces and one tab.

File: sort.js [hunk 1]
```diff
@@ -14,12 +14,18 @@ function swap(array, i, j) {
 
 function bubbleSort(a) {
   var n = a.length;
+  var sorted;
   for (var i = 0; i < n; i++) {
+  	sorted = true;
```
Comment (caspervonb) [inline sort.js]: This line contains tabs

File: sort.js [hunk 2]
```diff
@@ -14,12 +14,18 @@ function swap(array, i, j) {
...
     for (var j = 0; j < n - i - 1; j++) {
       if (test(a, j + 1, j) < 0) {
+	sorted = false;
```
Comment (caspervonb) [inline sort.js]: This line contains tabs.

**Review by caspervonb — CHANGES_REQUESTED** [review]
Replace tabs with spaces.

**Comment by rupav** [comment]
@caspervonb replacing 'tabs' with spaces worked fine.
Now its correct.
Can you tell me why tabs create problems?
Thanks :)

**Review by caspervonb — APPROVED** [review]

**Comment by caspervonb** [comment]
@rupav tabs don't have a set display size, so a tab can represent anywhere from 1-N spaces and different viewers (editors) have different settings.

Thanks, will merge in later ;)

**Comment by rupav** [comment]
Ok, thanks. :)

**Pull request merged**